src/averageAges.js

```diff
@@ -20,12 +20,28 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  let menAges;
+
+  century
+    ? menAges = people
+      .filter(({ sex }) => sex === 'm')
+      .filter(({ died }) => Math.ceil(died / 100) === century)
+      .map(({ born, died }) => died - born)
+    : menAges = people
+      .filter(({ sex }) => sex === 'm')
+      .map(({ born, died }) => died - born);
```

Comment:
nice job with destructuring 👍 
you do the same filtering twice

Reply:
rewritten